src/caching/github-actions-cache.ts

```diff
@@ -48,9 +49,80 @@ export class GitHubActionsCache implements Cache {
     this.#logger = logger;
   }
 
-  static create(
+  static async create(
     logger: Logger,
-  ): Result<GitHubActionsCache, {reason: 'invalid-usage'; message: string}> {
+  ): Promise<
+    Result<
+      GitHubActionsCache,
+      Omit<InvalidUsage, 'script'> | Omit<UnknownErrorThrown, 'script'>
+    >
+  > {
+    const custodianPort = process.env['WIREIT_CACHE_GITHUB_CUSTODIAN_PORT'];
+    if (custodianPort === undefined) {
+      if (
+        process.env['ACTIONS_RUNTIME_TOKEN'] !== undefined ||
+        process.env['ACTIONS_CACHE_URL'] !== undefined
+      ) {
+        console.warn(
+          '⚠️ Please upgrade to google/wireit@setup-github-cache/v2. ' +
```

Comment:
Should this be
`google/wireit@setup-github-actions-caching/v2`
or
`google/wireit@setup-github-cache/v2`
?